docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -1,30 +1,55 @@
 # MultiNetworkPolicies
-OVN-Kubernetes implements native support for
-[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
-an API providing
+
+## Introduction
+
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy) provides
 [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-features for secondary networks.
+features for secondary networks, and allow enhanced traffic security.
+
+## Motivation
+
+Using traffic restrictions based on user network configuration is a recommended practice. By defining policies that
+specify which services a workload can access, we mitigate supply chain attacks. Even if a component is compromised, it
+won’t be able to jeopardize other components of the system.
+
+## User-Stories
+
+As a cluster administrator, I want to enforce non-overridable admin network policies that must be adhered to by all user
+tenants in the cluster. This will secure my cluster’s network traffic.
+
+## User-case
+
+- Limit access to a database set outside the cluster, connected to VM and Pod workloads via localnet overlay network:
+  In this case example, only the VM has access to the database.
+  ![multi-homing-use-case-localnet](multi-homing-use-case-localnet.png)
+  ![micro-segmentation-use-case](micro-segmentation-use-case.png)
+
+## How to enable this feature on an OVN-Kubernetes cluster?
+
+In order to enable multi-networkPolicies on OVN-Kubernetes the `OVN_MULTI_NETWORK_ENABLE` config flag must be enabled
```

Comment:
nice; refer this also for the multi-homing feature itself.

Reply:
not sure I understood what you meant. changed. wdyt?